src/styles/index.scss

```diff
@@ -1,3 +1,255 @@
+$stars-size: 16px;
+
 body {
   margin: 0;
+  font-family: Roboto, 'Times New Roman', serif;
+  font-weight: 500;
+}
+
+.container {
+  --column-count: 4;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  justify-content: center;
+  gap: 46px 48px;
+}
+
+@media (min-width: 488px) {
+  .container {
+    --column-count: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .container {
+    --column-count: 3;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container {
+    --column-count: 4;
+  }
```

Comment:
The media query breakpoints do not follow a mobile-first approach. You should define styles for the smallest devices first, and then use min-width media queries to add styles as the viewport grows. This is in line with best practices for responsive design.